packages/react/src/components/UIShell/HeaderMenuButton.js

```diff
@@ -19,6 +19,8 @@ const HeaderMenuButton = ({
   'aria-label': ariaLabel,
   'aria-labelledby': ariaLabelledBy,
   className: customClassName,
+  customMenuIcon,
```

Comment:
Since we use `renderIcon` throughout the codebase, maybe `renderMenuIcon` would be more expected?

Reply:
Sure, renaming is done.